### PR TITLE
Update Getting Started section

### PR DESCRIPTION
**Pull request by HongW2019**
This PR updated Getting-Started section of documents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,45 +13,106 @@ of taking a lot of time constructing the cluster and platform. We target:
...
+| Python 3.8 | `pip install -U "cloudtik[aws] @ https://d30257nes7d4fq.cloudfront.net/downloads/cloudtik/cloudtik-0.9.0-cp38-cp38-manylinux2014_x86_64.whl" `     |
+| Python 3.7 | `pip install -U "cloudtik[aws] @ https://d30257nes7d4fq.cloudfront.net/downloads/cloudtik/cloudtik-0.9.0-cp37-cp37m-manylinux2014_x86_64.whl" `    |
+
+
+### 3. Authentication to Cloud Providers API
+
+After CloudTik is installed on your working machine, then you need to configure or log into your Cloud account to 
+gain access to cloud provider API on this machine.
+then you need 
+
```
Comment (jerrychenhf) [inline README.md:64]: "then you need"

What's this for? It seems to be a incomplete sentence.

File: README.md [hunk 2]
```diff
@@ -13,45 +13,106 @@ of taking a lot of time constructing the cluster and platform. We target:
...
+be configured and authenticated. *AWS Access Key ID* and *AWS Secret Access Key* can be found from the AWS guide of
+[managing access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
 
 ```
-cloudtik workspace create your-workspace-config.yaml
+$ aws configure
+AWS Access Key ID [None]: ...
+AWS Secret Access Key [None]: ...
+Default region name [None]: ...
+Default output format [None]:
```
Comment (jerrychenhf) [inline README.md:83]: This is too verbose. We don't need show these details. the above description as to aws-configure is enough.

File: README.md [hunk 3]
```diff
@@ -62,36 +123,139 @@ provider:
...
+provider:
+    type: aws
+    region: us-west-2
+    # S3 configurations for storage
+    aws_s3_storage:
+        s3.bucket: your_s3_bucket
+        s3.access.key.id: your_s3_access_key_id
+        s3.secret.access.key: your_s3_secret_access_key
+
+```
```
Comment (jerrychenhf) [inline README.md:169]: "Then fill out the `aws_s3_storage` field." -> This is just in the wrong way.
Does user know anything about these configuration? where is the configuration? NOTHING.
So we should not give this here.

File: README.md (hunk 2 above)
Comment (HongW2019) [reply README.md]: Moved this part to the next step.

Comment (HongW2019) [reply README.md]: Deleted this detail.

**Pull request merged**